src/components/MapboxMap/MapboxMap.tsx

```diff
@@ -52,7 +55,7 @@ export function MapboxMap({ children, ...rest }: MapboxMapProps) {
           mapboxAccessToken={MAPBOX_ACCESS_TOKEN}
           attributionControl={false}
           mapStyle={selectedStyleURI}
-          reuseMaps
```

Comment:
`reuseMaps` made it so that only the first map rendered would adjust its bounds, and I don't know that it actually made a difference in terms of performance or cost